### ISSUE TITLE
Add option for not showing pipelines at launch

### ISSUE THREAD

**Issue by bkuhlmann** [ccmenu/ccmenu2#15]
## Why

Hello. :wave: Would it be possible to disable the Pipelines window from showing up at launch? I'd like CCMenu to open within the menu bar without the extra window like it used to before the recent version release.

## How

Upgrade to Version 25.0.0 and then ensure you have the following set:

![2024-08-17_15-39-32-CCMenu](https://github.com/user-attachments/assets/151b8d31-e4f9-4c2d-8d62-21652a48b1f8)

The problem is that I'm getting the Pipelines (see screenshot below) window showing up upon application launch and think it'd be great if that was disabled by default (or maybe an option to disable)?

![2024-08-17_15-42-33-CCMenu](https://github.com/user-attachments/assets/f04643f6-4520-4632-bc60-87b972622166)

**Comment by erikdoe** [comment]
Agreed, the pipeline window shouldn't be visible when the application is opened, at least not by default. I've now added an option to the settings and its default is not to show the window when CCMenu is opened.
